protos/sources/vega/vega.proto

```diff
@@ -1621,3 +1621,13 @@ message BenefitTier {
   // Proportion of the referee's taker fees to be discounted
   string referral_discount_factor = 4;
 }
+
+message ActivityStreakBenefitTiers {
+  repeated ActivityStreakBenefitTier tiers = 1;
+}
+
+message ActivityStreakBenefitTier {
+  uint64 minimum_activity_streak = 1;
+  string reward_multiplier = 2;
+  string vesting_multiplier = 3;
+}
```

Comment:
Where are the doc strings?

Reply:
this is never exposed in an api, but I'll add them...